static_src/test/functional/overview_quicklook.spec.js

```diff
@@ -0,0 +1,50 @@
+
+import './global_setup';
+
+import OrgQuicklookElement from './pageobjects/org_quicklook.element';
+
+
+describe('Overview page', function () {
+  it('navigates to page', function () {
+    browser.url('/');
+  });
+
+  it('has a title', function () {
+    expect(browser.getTitle()).toBe('cloud.gov dashboard');
+  });
+
+  it('has a page header', function () {
+    const pageHeader = browser.element('.page-header-title');
```

Comment:
I have thoughts about how we should target elements for these tests. I wanted to prefix classes for tests with `test-` so that it's very clear to a developer that the element they're working with has a hook to functional tests. I found this greatly reduces regression in functional tests from HTML/element changes.